### PR TITLE
CLOP-12 continue to manage Athena in TDR mgmt account

### PR DESCRIPTION
**Pull request by paulschwarzenberger**

### CODE REVIEW

File: moved.tf [hunk 1]
```diff
@@ -0,0 +1,24 @@
...
+
+moved {
+  from = module.athena_s3.aws_s3_bucket_server_side_encryption_configuration.encryption_configuration
+  to   = module.athena_s3[0].aws_s3_bucket_server_side_encryption_configuration.encryption_configuration
+}
+
+moved {
+  from = module.athena_s3.aws_s3_bucket_versioning.versioning
+  to   = module.athena_s3[0].aws_s3_bucket_versioning.versioning
+}
```
Comment (TomJKing) [inline moved.tf:24]: Missing line break at end of file.

**Review by TomJKing — COMMENTED** [review]
Looks good. Thanks for doing this. Just one very minor code format issue.

File: moved.tf (hunk 1 above)
Comment (paulschwarzenberger) [reply moved.tf]: thanks, updated

**Review by TomJKing — APPROVED** [review]
Looks good. Thanks

**Pull request merged**